public/js/controllers/game.js

```diff
@@ -13,7 +13,7 @@ angular.module('mean.system')/* eslint-disable-line */
       $scope.pickedCards = [];
       // let makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
       // $scope.makeAWishFact = makeAWishFacts.pop();
-
+      $scope.name = 'ello bae';
```

Comment:
This line doesn't seem to be useful to be used anywhere.
Can it be possibly removed?